src/utils/hooks/useFetchPublication.tsx

```diff
@@ -0,0 +1,37 @@
+import { useState, useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import Axios from 'axios';
+
+import { setNotification } from '../../redux/actions/notificationActions';
+import { NotificationVariant } from '../../types/notification.types';
+import { Publication } from '../../types/publication.types';
+import { getPublication } from '../../api/publicationApi';
+
+const useFetchPublication = (identifier: string): [Publication | undefined, boolean] => {
```

Comment:
denne kan sikkert brukes flere plasser enn bare `PublicationPage` ?

Reply:
kan sikkert ta den i bruk for PublicationForm i samme slengen ja!

Reply:
tar du med det i denne PRen eller?

Reply:
jepp!

Reply:
Det var litt mer knot enn jeg trodde, siden vi gjør litt andre ting også i den useEffecten. Lager en egen sak for dette likevel: NP-968